citadel-hive/src/controllers/instance.ts

```diff
@@ -106,12 +106,15 @@ class InstanceController extends commonControllers.ApplicationController {
   // POST /instances/:name/stop
   async stop(ctx: Context) {
     const { name } = ctx.params as { name: string };
+    const { drone } = ctx.request.body as { drone: string };
```

Comment:
As mentioned previously, this should be fetched from the database with the user who makes the request and the name of the instance :)